fuzz/ocsp_http.cc

```diff
@@ -0,0 +1,24 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include <openssl/err.h>
+#include <openssl/mem.h>
+#include <openssl/ocsp.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *buf, size_t len) {
+  bssl::UniquePtr<BIO> bio(BIO_new_mem_buf(buf, len));
+  OCSP_REQ_CTX *req_ctx = OCSP_REQ_CTX_new(bio.get(), 0);
+  OCSP_RESPONSE *ocsp_response;
+
+  // OCSP_sendreq_nbio parses the HTTP response in the BIO.
+  if(OCSP_sendreq_nbio(&ocsp_response, req_ctx) > 0) {
```

Comment:
Is this attempting to actually send the request?

Reply:
Yeah this hadn't been fuzzing what I initially wanted. I've updated it to use a dummy request, so we can properly get to parsing the OCSP http response. Also updated to fuzz `OCSP_REQ_CTX_nbio` instead, which is directly calling the `parse_http_line` static function.